### DISCUSSION TITLE
How do I enable EMA in dreambooth training?

### DISCUSSION THREAD

**Discussion by nayan-dhabarde** [huggingface/diffusers#6746] [Q&A]
I have checked a lot of blogs saying enabling EMA has yielded people better results, I have searched through the documentation but couldn't find anything on EMA

**Comment by sayakpaul** [comment]
EMA with LoRA DreamBooth provides better results?

Could you provide some resources confirming this? 

If you don't want to do LoRA on DreamBooth, then refer to the https://github.com/huggingface/diffusers/blob/main/examples/text_to_image/train_text_to_image.py script as it provides a nice reference.
